request help: I cann't build openresty with apisix-nginx-module

### Issue description

I build the openresty used the sh file like this https://raw.githubusercontent.com/api7/apisix-build-tools/master/build-apisix-openresty.sh .
this is the out put error:
#######################################################
-L/tmp/tmp.0wvL1VhoXA/openresty-1.19.3.2/build/luajit-root/usr/local/openresty/luajit/lib -L/tmp/tmp.0wvL1VhoXA/openresty-1.19.3.2/build/luajit-root/usr/local/openresty/luajit/lib -Wl,-rpath,/usr/local/openresty/luajit/lib -Wl,--require-defined=pcre_version -Wl,-E -Wl,-E -ldl -lpthread -lpthread -lcrypt -L/tmp/tmp.0wvL1VhoXA/openresty-1.19.3.2/build/luajit-root/usr/local/openresty/luajit/lib -lluajit-5.1 -lm -ldl -L/tmp/tmp.0wvL1VhoXA/openresty-1.19.3.2/build/luajit-root/usr/local/openresty/luajit/lib -lluajit-5.1 -lm -ldl -lstdc++ -lpcre -lssl -lcrypto -ldl -lpthread -lz \
-Wl,-E
objs/addon/src/ngx_http_apisix_module.o：在函数‘ngx_http_apisix_x509_copy’中：
/tmp/tmp.0wvL1VhoXA/openresty-1.19.3.2/../apisix-nginx-module/src/ngx_http_apisix_module.c:96：对‘X509_up_ref’未定义的引用
objs/addon/src/ngx_http_apisix_module.o：在函数‘ngx_http_apisix_upstream_set_cert_and_key’中：
/tmp/tmp.0wvL1VhoXA/openresty-1.19.3.2/../apisix-nginx-module/src/ngx_http_apisix_module.c:187：对‘EVP_PKEY_up_ref’未定义的引用
collect2: 错误：ld 返回 1
make[2]: *** [objs/nginx] 错误 1
make[2]: 离开目录“/tmp/tmp.0wvL1VhoXA/openresty-1.19.3.2/build/nginx-1.19.3”
make[1]: *** [build] 错误 2
make[1]: 离开目录“/tmp/tmp.0wvL1VhoXA/openresty-1.19.3.2/build/nginx-1.19.3”
make: *** [all] 错误 2
#######################################################
who can I fix it ,thank's.

### Environment

- apisix version (cmd: `apisix version`):2.6
- OS (cmd: `uname -a`):linux 3.10.0-1127.19.1.el7.x86_64
- OpenResty / Nginx version (cmd: `nginx -V` or `openresty -V`):
- etcd version, if have (cmd: run `curl http://127.0.0.1:9090/v1/server_info` to get the info from server-info API):
- apisix-dashboard version, if have:
- luarocks version, if the issue is about installation (cmd: `luarocks --version`):

--- a/docs/en/latest/how-to-build.md
+++ b/docs/en/latest/how-to-build.md
@@ -237,7 +237,9 @@ Content-Type: text/html
 
 ## Step 6: Build OpenResty for Apache APISIX
 
-Some features require additional NGINX modules to be introduced into OpenResty. If you need these features, you can build OpenResty with [this script](https://raw.githubusercontent.com/api7/apisix-build-tools/master/build-apisix-openresty.sh).
+Some features require additional NGINX modules to be introduced into OpenResty.
+If you need these features, you can build the APISIX OpenResty.
+You can refer to the source of [api7/apisix-build-tools](https://github.com/api7/apisix-build-tools) for how to set up the build environment and build the APISIX OpenResty.
 
 ## Step 7: Add Systemd Unit File for Apache APISIX
 

--- a/docs/zh/latest/how-to-build.md
+++ b/docs/zh/latest/how-to-build.md
@@ -235,7 +235,9 @@ Content-Type: text/html
 
 ## 步骤6：为 Apache APISIX 构建 OpenResty
 
-有些功能需要引入额外的 NGINX 模块到 OpenResty 当中。如果您需要这些功能，你可以用[这个脚本](https://raw.githubusercontent.com/api7/apisix-build-tools/master/build-apisix-openresty.sh)构建 OpenResty。
+有些功能需要引入额外的 NGINX 模块到 OpenResty 当中。
+如果您需要这些功能，您可以构建 APISIX OpenResty。
+您可以根据[api7/apisix-build-tools](https://github.com/api7/apisix-build-tools)里面的代码，配置自己的构建环境，并完成 APISIX OpenResty 的构建。
 
 ## 步骤7：为 Apache APISIX 添加 systemd 配置文件
 

